Source/Game.cs

```diff
@@ -40,22 +40,42 @@ private enum TransitionStep
 	public const string GamePath = "Celeste64";
     // ModloaderCustom
     public const string GameTitle = "Celeste 64: Fragments of the Mountain + Fuji Mod Loader";
-	public const int Width = 640;
-	public const int Height = 360;
 	public static readonly Version GameVersion = typeof(Game).Assembly.GetName().Version!;
 	public static readonly string VersionString = $"Celeste 64: v.{GameVersion.Major}.{GameVersion.Minor}.{GameVersion.Build}";
 	public static string LoaderVersion { get; set; } = "";
+	
+	public const int DefaultWidth = 640;
+	public const int DefaultHeight = 360;
+	
+	public static event Action OnResolutionChaned;
+	
+	private static float _resolutionScale = 1.0f;
+	public static float ResolutionScale
+	{
+		get => _resolutionScale;
+		set
+		{
+			if (_resolutionScale == value)
+				return;
+			
+			_resolutionScale = value;
+			OnResolutionChaned.Invoke();
+		}
+	}
```

Comment:
It seems like this getting set is the main trigger for everything kicking off, but at what point does this actually get set? I don't think I'm seeing any calls in the code to set this. Are we planning to have this get set by mods or something?